pkg/services/control/server/helpers.go

```diff
@@ -0,0 +1,20 @@
+package control
+
+import "github.com/nspcc-dev/neofs-node/pkg/local_object_storage/shard"
+
+func (s *Server) getShardIDList(raw [][]byte) []*shard.ID {
```

Comment:
if there is some bug in the `neofs-cli` or the req has been sent by another tool and we get an empty list of the shards to evacuate, do we... evacuate all of them?

Reply:
Yes, but it should be simple -- we check the length

Reply:
it scares me, i dunno why

but ok